### ISSUE TITLE
paddle在MNIST数据集上损失一直没有下降

### ISSUE THREAD

**Issue by ghost** [PaddlePaddle/Paddle#34469]
PaddlePaddle版本：2.1.1
Python版本： 3.8.0
系统：Windows 10

```python
import paddle
from paddle import nn
from paddle import optimizer
from paddle.vision import datasets
from paddle.io import Dataset, DataLoader
from paddle.vision import transforms as T
import numpy as np

transforms = T.Compose([T.ToTensor(), T.Normalize(mean=[127.5], std=[127.5])])

train_data = datasets.MNIST(mode='train', download=True, transform=transforms)
test_data = datasets.MNIST(mode='test', download=True, transform=transforms)

class MyDataset(Dataset):
    
    def __init__(self, train=True):
        self.train = train
        if train:
            self.data = train_data
        else:
            self.data = test_data
    
    def __getitem__(self, index):
        return self.data[index]
    
    def __len__(self):
        return len(self.data)
    

BATCH_SIZE = 128
train_loader = DataLoader(MyDataset(train=True), batch_size=BATCH_SIZE, shuffle=True)
test_loader = DataLoader(MyDataset(train=False), batch_size=BATCH_SIZE, shuffle=False)
net = nn.Sequential(
    nn.Flatten(),
    nn.Linear(784, 512),
    nn.ReLU(),
    nn.Linear(512, 256),
    nn.ReLU(),
    nn.Linear(256, 128),
    nn.ReLU(),
    nn.Linear(128, 10)
)
net.train()
optim = optimizer.SGD(learning_rate=1e-6, parameters=net.parameters())
criterion = paddle.nn.CrossEntropyLoss()


for epoch in range(5):
    for i, (x, y) in enumerate(train_loader):
        y = paddle.to_tensor(y, stop_gradient=True)
        y_pred = net(x)
        loss = criterion(y_pred, y)
        loss.backward()
        optim.step()
        optim.clear_grad()
        if (i + 1) % 50 == 0:
            print(f'loss: {loss.numpy()[0]:.2f}')
```

结果（一直停留在2.3左右，而官方文档https://www.paddlepaddle.org.cn/documentation/docs/zh/guides/02_paddle2.0_develop/01_quick_start_cn.html 按照paddle.Model的方式却没事）：
![image](https://user-images.githubusercontent.com/53207383/127332643-73e4d493-839e-4d01-af93-97ef021064c0.png)

**Comment by ghost** [comment]
已经解决，将`transforms = T.Compose([T.ToTensor(), T.Normalize(mean=[127.5], std=[127.5])])`改为`transforms = T.Compose([T.Normalize(mean=[127.5], std=[127.5])])`即可
